db/migrate/20180208161547_add_time_limit_type_to_correspondence_types.rb

```diff
@@ -0,0 +1,14 @@
+class AddTimeLimitTypeToCorrespondenceTypes < ActiveRecord::Migration[5.0]
+  def up
+    CorrespondenceType.find_by(abbreviation: 'FOI')
+                      &.update time_limit_type: 'business_days'
+    CorrespondenceType.find_by(abbreviation: 'GQ')
```

Comment:
I think we should get rid of GQ in the seeder.

Reply:
This code here only updates the record if it exists, I think it would be worth it to leave it here ... we could remove it from `db/seeders/correspondence_type_seeder.rb`, do we need to also write a migration to remove it from any existing db?